### PR TITLE
Button branch

### PR DESCRIPTION
**Pull request by Shivani-1524**
I've added the button component to my library it includes solid buttons, outlined buttons, icon buttons.

### CODE REVIEW

**Review by kaustubh-jsr — COMMENTED** [review]
Looks good to me. This could be an oversight, but in the preview deployed all code snippets, barring the first two, show same code snippets you might want to update that. Rest looks fine.

File: Components/Button/button.css [hunk 1]
```diff
@@ -3,33 +3,182 @@
...
+}
+.icon-btn {
+  padding: 0px;
+}
+.icon-btn.danger-icon {
+  color: var(--red-color);
+}
+.icon-btn.primary-icon {
+  width: 33px;
+  height: 33px;
```
Comment (rdthetha) [inline Components/Button/button.css]: you can use rem instead of px

File: documentation.html [hunk 2]
```diff
@@ -56,11 +56,109 @@
...
+                            </div>
+                        </div>
+                        <div class="code-example">
+                            <iframe class="code-snip"
+                                src="https://carbon.now.sh/embed?bg=rgba%28189%2C213%2C234%2C0%29&t=vscode&wt=sharp&l=auto&width=859&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=18px&lh=144%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2520primary-btn%2522%253EPrimary%2520Button%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520secondary-btn%2522%253ESecondary%2520Button%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520success-btn%2522%253ESuccess%2520Button%253C%252Fbutton%253E%2520%250A%253Cbutton%2520class%253D%2522btn%2520warning-btn%2522%253EWarning%2520Button%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520danger-btn%2522%253EError%2520Button%253C%252Fbutton%253E"
+                                style="width: 677px; height: 204px; border:0; transform: scale(1); overflow:hidden;"
+                                sandbox="allow-scripts allow-same-origin">
```
Comment (rdthetha) [inline documentation.html:83]: you can remove the inline styling here and shift to the css file. Also the iframe are same for the examples

File: documentation.html [hunk 3]
```diff
@@ -56,11 +56,109 @@
...
+                                    <i class="fas fa-bookmark"></i>
+                                </button>
 
+                            </div>
+                        </div>
+                        <div class="code-example">
+                            <iframe class="code-snip"
+                                src="https://carbon.now.sh/embed?bg=rgba%28189%2C213%2C234%2C0%29&t=vscode&wt=sharp&l=auto&width=859&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=18px&lh=144%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2520primary-btn%2522%253EPrimary%2520Button%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520secondary-btn%2522%253ESecondary%2520Button%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520success-btn%2522%253ESuccess%2520Button%253C%252Fbutton%253E%2520%250A%253Cbutton%2520class%253D%2522btn%2520warning-btn%2522%253EWarning%2520Button%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520danger-btn%2522%253EError%2520Button%253C%252Fbutton%253E"
+                                style="width: 677px; height: 204px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (nikhiljugale007) [inline documentation.html]: You can move this inline styling of iframe to a common class.

File: Components/FloatingButton/floatingbtn.css [hunk 4]
```diff
@@ -18,7 +19,7 @@
 
 .floating-btn i {
   font-size: 20px;
```
Comment (nikhiljugale007) [inline Components/FloatingButton/floatingbtn.css]: We should use rem or em for font size.

**Comment by nikhiljugale007** [comment]
Suggested some changes in code.  Other than that UI and Code look good.

**Comment by Shivani-1524** [comment]
> Looks good to me. This could be an oversight, but in the preview deployed all code snippets, barring the first two, show same code snippets you might want to update that. Rest looks fine.

yes, corrected it. thanks for pointing it out.

File: Components/Button/button.css (hunk 1 above)
Comment (Shivani-1524) [reply Components/Button/button.css]: yes, will change it

File: documentation.html (hunk 2 above)
Comment (Shivani-1524) [reply documentation.html:83]: okay will change it

File: documentation.html (hunk 3 above)
Comment (Shivani-1524) [reply documentation.html]: They have small differences between each code example. does it mean I have to create a class for each code snippet?

File: Components/FloatingButton/floatingbtn.css (hunk 4 above)
Comment (Shivani-1524) [reply Components/FloatingButton/floatingbtn.css]: okay, will change it

**Comment by Shivani-1524** [comment]
> Suggested some changes in code. Other than that UI and Code look good.

Thanks a lot for the review and the compliment !!

**Pull request merged**